notebooks/brian_long_smFISH.ipynb

```diff
@@ -0,0 +1,131 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Reproduce Brian Long's results with Starfish\n",
+    "\n",
+    "The `brian_long_smFISH.zip` file needed to follow along with this notebook can be downloaded [here]()\n",
+    "\n",
+    "This notebook walks through a work flow that reproduces the smFISH result for one field of view using the starfish package. \n",
+    "It assumes that you have unzipped `brian_long_smFISH.zip` in the same directory as this notebook. Thus, you should see:\n",
+    "\n",
+    "raw/\n",
+    "brian_long_smFISH.ipynb"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from starfish.io import Stack\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# package this up same as before\n",
+    "experiment_json = os.path.expanduser('./brian_long_smFISH/experiment.json')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "s = Stack()\n",
+    "s.read(experiment_json)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# brian's data is uint, but starfish loads it as floats. Cast it to int here, which does not cause any loss of precision. \n",
```

Comment:
we should enforce starfish loading the data as uint16 -- at least that's what we request in our pipeline spec.

Reply:
Agree. #149